gammapy/irf/energy_dispersion.py

```diff
@@ -541,6 +545,291 @@ def gauss_energy_dispersion_matrix(ebounds, sigma=0.2):
 
     return pdf_matrix
 
-    #hdu_list = np_to_rmf(rm, ea_erange, ea_erange, 1E-5,
+    # hdu_list = np_to_rmf(rm, ea_erange, ea_erange, 1E-5,
     #                     telescope=telescope, instrument=instrument)
-    #return hdu_list
+    # return hdu_list
+
+
+class EnergyDispersion2D(object):
+
+    """Offset-dependent energy dispersion matrix.
+
```

Comment:
Document `Parameters` here?
Move plotting examples to the plot method docstrings?